src/Deals.php

```diff
@@ -101,13 +101,23 @@ public function getBySearch($search, $limit = false, $offset = 0)
 		return $this->formatDealsForReturn($deals);
 	}
 
-	public function getBaseQuery($limit = false, $offset = 0)
+	public function getBaseQuery($limit = false, $offset = 0, $orderField = 'fRating', $desc = true)
 	{
 		$query = DB::table('deals')
 					->select(DB::raw('fmtc_merchants.*, fmtc_deals.*'))
 					->join('merchants', 'deals.nMerchantID', '=', 'merchants.nMerchantID')
-					->where('dtEndDate', '>=', date('Y-m-d H:i:s'))
-					->orderBy('fRating', 'desc');
+					->where('dtEndDate', '>=', date('Y-m-d H:i:s'));
+		
+		$order = 'asc';
+		if($desc) {
+			$order = 'desc';
+		}
+		
```

Comment:
this should be re-written using a ternary operator: $order = ($desc) ? 'desc' : 'asc';
- turns 4 lines of a clunky conditional and turns it into 1 line of a readable assignment.